Bixal/move.mil
# Feature/156844285/parse entitlements file

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Address pivotal ticket #156844285
- Writes zip5.json

## Testing

To verify the changes proposed in this pull request…

1. Execute `docker-compose run php drupal parser` with the entitlements option.
1. Verify entitlements.json is written at `web/sites/default/files/tools/data`.

## Screenshots
```
☁  move3 [feature/156844285/parse_zip5_file] ⚡ docker-compose run php drupal parser
 Which is the file name to be parsed? [zip3]:
 > entitlements
 Parsing /var/www/html/lib/data/entitlements.yml...
 File parsed
```
![screen shot 2018-04-23 at 3 39 46 pm](https://user-images.githubusercontent.com/37001835/39149289-9ca638ec-470c-11e8-9eb9-50449b6f897c.png)
